maven-release-manager/src/main/java/org/apache/maven/shared/release/DefaultReleaseManager.java

```diff
@@ -726,7 +732,10 @@ private void logInfo( ReleaseResult result, String message )
 
     private void captureException( ReleaseResult result, ReleaseManagerListener listener, Exception e )
     {
-        updateListener( listener, e.getMessage(), ERROR );
+        if ( listener != null )
+        {
+            listener.error( e.getMessage() );
```

Comment:
Pity that exception object is lost

Reply:
it's just for a basic log line: immediately, the exception will be thrown wrapped in a MojoException

Reply:
Makes sense!